app/views/layouts/registration_attractive.html.erb

```diff
@@ -0,0 +1,27 @@
+<!doctype html>
+<html lang="en" class="govuk-template">
+  <% content_for :head do %>
+    <%= javascript_pack_tag 'govuk/error_summary', 'data-turbolinks-track': 'reload', async: true %>
+  <% end %>
+
+  <%= render "sections/head" %>
+  <%= analytics_body_tag class: "govuk-template__body govuk-body", data: { controller: "video link scroll", "link-target": "content" }, id: "body" do %>
+
+    <%= render HeaderComponent.new %>
+
+    <main role="main" id="main-content" class="attractive-layout">
+      <section class="signup-container">
+
+        <%= image_pack_tag 'content/hero-images/0025.jpg', class: 'left-img hide-on-mobile hide-on-tablet' %>
```

Comment:
Can we add alt tags to these?

Reply:
@leoapost

Reply:
I'd create empty alt tags for these ones, `alt: ""`, as they're purely for decoration and we wouldn't want the screen reader to distract the user from the form.

Annoyingly, if there's no alt tag at all the screen reader will just read out the filename.

Reply:
I just pushed a commit with the empty `alt` attributes.